### PR TITLE
fix lua-tests errors 

### PR DESCRIPTION
**Pull request by PatriceJiang**
ref https://github.com/cocos2d/cocos2d-x/issues/19630

This PR is intended to be merged after https://github.com/cocos2d/bindings-generator/pull/250

### CODE REVIEW

File: cocos/scripting/lua-bindings/manual/lua_module_register.cpp [hunk 1]
```diff
@@ -44,9 +44,9 @@ int lua_module_register(lua_State* L)
     register_cocostudio_module(L);
     register_ui_module(L);
  //TODO minggo
-//    register_extension_module(L);
+    register_extension_module(L);
```
Comment (minggo) [inline cocos/scripting/lua-bindings/manual/lua_module_register.cpp:46]: can remove the TODO now

File: tests/lua-tests/src/ParticleTest/ParticleTest.lua [hunk 2]
```diff
@@ -29,7 +29,7 @@ end
 
 local function nextAction()
 	SceneIdx = SceneIdx + 1
-    SceneIdx = math.mod(SceneIdx, MAX_LAYER)
+    SceneIdx = math.fmod(SceneIdx, MAX_LAYER)
```
Comment (minggo) [inline tests/lua-tests/src/ParticleTest/ParticleTest.lua:32]: why use `fmod`?

File: tools/tolua/cocos2dx_backend.ini [hunk 3]
```diff
@@ -0,0 +1,62 @@
+[cocos2d-backend]
+# the prefix to be added to the generated functions. You might or might not use this in your own
+# templates
+prefix = cocos2dx_backend
+
+# create a target namespace (in javascript, this would create some code like the equiv. to `ns = ns || {}`)
+# all classes will be embedded in that namespace
+target_namespace = ccbackend
```
Comment (minggo) [inline tools/tolua/cocos2dx_backend.ini]: use `ccb` for short

File: tests/lua-tests/src/ParticleTest/ParticleTest.lua (hunk 2 above)
Comment (PatriceJiang) [reply tests/lua-tests/src/ParticleTest/ParticleTest.lua:32]: luajit has been upgraded from  `2.1.0-beta2` to `2.1.0-beta3` in https://github.com/cocos2d/cocos2d-x-3rd-party-libs-bin/pull/337/files

https://github.com/elikosan/luajit-2.1/blob/master/doc/luajit.html
> Remove old Lua 5.0 compatibility defines

**Pull request merged**